src/plugins/graph/components/scatterdots.tsx

```diff
@@ -179,7 +179,7 @@ export const ScatterDots = function ScatterDots(props: PlotProps) {
       v2Scale = layout.getAxisScale("rightNumeric") as ScaleNumericBaseType,
       numExtraPrimaryBands = dataConfiguration?.numRepetitionsForPlace('bottom') ?? 1,
       numExtraSecondaryBands = dataConfiguration?.numRepetitionsForPlace('left') ?? 1,
-      numberOfPlots = dataConfiguration?.numberOfPlots || 1,
+      numberOfPlots = dataConfiguration?.numberOfPlots || 0,
```

Comment:
This seems curious to me -- I would have thought that assuming that there's always a plot would simplify things.